clients/unshaded/src/main/java/tachyon/client/TachyonByteBuffer.java

```diff
@@ -50,7 +50,7 @@
   }
 
   /**
-   * Closes the TachyonByteBuffer, here it is synchronized.
+   * Closes the {@code TachyonByteBuffer}, here it is synchronized.
```

Comment:
See my points mentioned above. In that case it would be "```{@code ...}``` is used in the method documentation if the class itself is referenced."